### ISSUE TITLE
Different accuracy & cannot evaluation

### ISSUE THREAD

**Issue by JaehunKim-DeepLearning** [pytorch/fairseq#2951]
Hi

i ran the finetuned  base model in 10.1 version for Korean and, i got the 15% cer about my validation set in trainning
also i ran the new repo and got the 10% cer
why different result? 
i checked the hyper parameters 
and i couldn't find any different 
can i know changed that parameters?
about 10.1 version and new repo

**Comment by myleott** [comment]
What command/data are you using? 0.10.1 is very similar to the current master, so it’s surprising if you get different results...
